src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxNode.cs

```diff
@@ -460,6 +460,26 @@ public new SyntaxTrivia FindTrivia(int position, bool findInsideTrivia = false)
 
 #region SyntaxNode members
 
+        private protected override bool IsSimilarToCore(SyntaxNode other)
+        {
+            int kindGroup(SyntaxNode node)
```

Comment:
>kindGroup [](start = 16, length = 9)

nit: we tend to put local functions last. I went ahead and adjusted to save time